SQL/New_patches/2019_07_02_Add_Edit_DoB_Permissions.sql

```diff
@@ -0,0 +1,5 @@
+INSERT INTO permissions (code, description, categoryID) VALUES
+    ("candidate_dob_edit","Edit dates of birth",2);
+
+INSERT INTO user_perm_rel VALUES
+    ((SELECT ID from users WHERE UserID='admin'),(SELECT permID FROM permissions WHERE code='candidate_dob_edit'));
```

Comment:
userID = 'admin' not correct, in my case, I used 'lorisadmin' as userID. In other people's case they may used a different userID. Consider use something not varies across people.

Reply:
@driusan correct me if I'm wrong but `admin` exists in the default install and is always referred to when creating new permissions no ?

should we rely on `ID=1` instead ? that doesnt seem like a better idea

Reply:
`admin` does exist in the default install afaik. 

In any case I don't think we need this line. The `admin` equivalent will probably have the superuser permission. Other admin-level users can probably be granted this permission on case-by-case basis.

Reply:
admin exists in the default install but may have been renamed or deleted on various installs, or installs that happened before that default was created.

But doesn't admin have highlander anyways?

Reply:
Installing the Database - 2 of 2
[2. Fill in \<loris-admin-username\>, \<loris-admin-passwrd\>; e.g. admin](https://github.com/aces/Loris/wiki/Hosting-the-Database-Myself)
The installing instruction used "e.g. admin", people may randomly choose their user names. Not sure whether that is something to consider.

Reply:
alright, removed-